dev.edfloreshz.Done.Devel.json

```diff
@@ -0,0 +1,89 @@
+{
+    "app-id" : "dev.edfloreshz.Done.Devel",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "done",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+	        "CARGO_HOME" : "/run/build/done/cargo",
+          "PROTOC": "/app/bin/protoc"
+	    }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "protobuf-compiler",
+            "buildsystem": "simple",
+            "build-commands": [
+                "unzip protobuf.zip -d protobuf",
+                "mkdir -p $FLATPAK_DEST/bin/",
+                "chmod 755 protobuf/bin/protoc",
+                "cp protobuf/bin/protoc $FLATPAK_DEST/bin/",
+                "cp -dr protobuf/include $C_INCLUDE_PATH/"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://github.com/protocolbuffers/protobuf/releases/download/v21.12/protoc-21.12-linux-x86_64.zip",
+                    "dest-filename": "protobuf.zip",
+                    "sha256": "3a4c1e5f2516c639d3079b1586e703fc7bcfa2136d58bda24d1d54f949c315e8"
+                }
+            ]
+        },
+        {
+            "name": "libprotobuf-dev",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x libprotobuf-dev.deb",
+                "tar -xf data.tar.xz",
+                "mkdir -p $FLATPAK_DEST/lib/pkgconfig",
+                "cp -dr usr/lib/x86_64-linux-gnu/pkgconfig/* $FLATPAK_DEST/lib/pkgconfig"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "http://ftp.us.debian.org/debian/pool/main/p/protobuf/libprotobuf-dev_3.21.12-1+b1_amd64.deb",
+                    "dest-filename": "libprotobuf-dev.deb",
+                    "sha256": "cccdf6cb30a00c5c72543dbffe78059b23c7819ceeb9d26818319bb7c5b2ed2e"
+                }
+            ]
+        },
+        {
+            "name" : "done",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dbuildtype=release",
+                "-Dprofile=development"
```

Comment:
I doubt this should be development profile....

Reply:
This is the development version of the app.